### PR TITLE
Delete rejected match.

### PR DESCRIPTION
**Pull request by lyw0149**
1. Delete rejected match item.
2. Delete unnecessary console.log(). 

Please review code. @nlpenguin @ethkim @heartjigi @KaylaKwon @ace0625

### CODE REVIEW

File: src/server/controllers/match.controller.js [hunk 1]
```diff
@@ -158,13 +158,24 @@ function findMentorActivity(mentor_id) {
...
   if (req.session._id) {
-    Match.update({ _id: req.body.match_id }, { status: req.body.option, response_date: Date.now() }, (err) => {
-      if (err) {
-        res.status(400).json({ err_point: matchCallback.ERR_MONGOOSE, err: err });
-      } else {
-        res.status(200).json({ msg: matchCallback.SUCCESS_RESPONSE });
-      }
-    });
+    ///todo: Validiate  match_id & option params (Luke Lee)
+    if (req.body.option === '0') {
```
Comment (ifndefdeadmau5) [inline src/server/controllers/match.controller.js]: Use constant instead of '0'

**Comment by ifndefdeadmau5** [comment]
:shipit:

File: src/server/controllers/match.controller.js (hunk 1 above)
Comment (lyw0149) [inline src/server/controllers/match.controller.js]: Checked.

**Comment by ifndefdeadmau5** [comment]
LGTM

**Pull request merged**